site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -260,35 +252,23 @@ Agones supports Autopilot clusters in [alpha]({{<ref "/docs/Guides/feature-stage
     Replace the following:
 
 * **CLUSTER_NAME**: the name of your cluster.
+* **COMPUTE_REGION**: the GCP region to create the cluster in
 * **RELEASE_CHANNEL**: one of
-  `rapid`, `regular`, or `stable`. The default is `regular`. To set a specific
+  `rapid`, `regular`, or `stable`, chosen in the previous step. The default is `regular`. To set a specific
   available version in the release channel, use the
   `--cluster-version=[VERSION]` flag.
 
 ## Setting up cluster credentials
 
-Finally, let's tell `gcloud` that we are speaking with this cluster, and get auth credentials for `kubectl` to use.
+`gcloud container clusters create` configurates credentials for `kubectl` automatically. If you ever lose those, run:
 
 ```bash
-gcloud config set container/cluster [CLUSTER_NAME]
-gcloud container clusters get-credentials [CLUSTER_NAME]
+gcloud container clusters get-credentials [CLUSTER_NAME] --region=[COMPUTE_REGION]
 ```
 
 [cluster]: https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture
 [vms]: https://cloud.google.com/compute/docs/instances/
 
-
-{{< alert title="Note" color="info">}}
-Before planning your production GKE infrastructure, it is worth reviewing the
-[different types of GKE clusters that can be created](https://cloud.google.com/kubernetes-engine/docs/concepts/types-of-clusters),
-such as Zonal or Regional, as each has different reliability and cost values, and ensuring this aligns with your
-Service Level Objectives or Agreements.
-
-This is particularly true for a zonal GKE control plane, which can go down temporarily to adjust for cluster resizing,
-[automatic upgrades](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-upgrades) and
-[repairs](https://cloud.google.com/kubernetes-engine/docs/concepts/maintenance-windows-and-exclusions#repairs).
-{{< /alert >}}
-
 ## Next Steps
 
 ### GKE Standard
```

Comment:
In the autopilot section below it says "These installation instructions apply to Agones 1.30." I think they also apply to 1.31 and unless we close https://github.com/googleforgames/agones/issues/3098 in the next day or so they will apply to 1.32 as well. Can you adjust that to specify the range and also mention that support for GKE autopilot was added in 1.30?

Reply:
Do we need to specify when the support was added?

Changed it to 1.30+.